src/main/java/com/exactpro/th2/common/schema/factory/CommonFactory.java

```diff
@@ -262,6 +264,7 @@ public static CommonFactory createFromArguments(String... args) {
         Option namespaceOption = new Option(null, "namespace", true, null);
         Option boxNameOption = new Option(null, "boxName", true, null);
         Option contextNameOption = new Option(null, "contextName", true, null);
+        Option dictionariesOption = new Option(null, "dictionaries", true, null);
```

Comment:
Probably, we need to set the `setArgs(Option.UNLIMITED_VALUES)` to use multiple values for the option instead of one option's value. I am not sure yet